yt/yt/server/master/cell_master/program.h

```diff
@@ -102,6 +110,10 @@ class TCellMasterProgram
             THROW_ERROR_EXCEPTION("Option 'build-snapshot' can only be used with 'validate-snapshot' or 'replay-changelog'");
         }
 
+        if (PrintCompatibilityInfo_) {
```

Comment:
Below we have a different "HandleSomething" pattern. Can we follow it here as well?

Reply:
Would it be enough to change method to `bool HandleCompatibilityInfo()`? Or do you want some extendable generic class for compatibility info which is enriched with reign for cell-master?